packages/table/src/table.tsx

```diff
@@ -590,6 +595,52 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     /**
+     * Scroll the viewport such that the target region ends up in the top-left corner. If `animated`
+     * is `true`, the table will animate the scroll. This can help users better understand how the
+     * table has shifted.
+     *
+     * Scrolling will behave for each region cardinality as follows:
+     *
+     *   CELLS:
+     *   Scroll the top-left cell in the target region to the top-left corner of the viewport.
+     *
+     *   FULL_ROWS:
+     *   Scroll the top-most row in the target region to the top of the viewport.
+     *
+     *   FULL_COLUMNS:
+     *   Scroll the left-most column in the target region to the left side of the viewport.
+     *
+     *   FULL_TABLE:
+     *   Scroll the top-left cell to the top-left corner of the viewport.
```

Comment:
can cut lines in a third by formatting as list.

```
- `CELLS`: words words
- `FULL_ROWS`: more words
```

Reply:
One of them was too long to fit and line-wrapped; this format takes more lines but keeps everything looking perfectly consistent. I'll see if I can whittle the words down and go back to a list.